minisom.py

```diff
@@ -170,11 +238,17 @@ def euclidean(x, w):
         random_seed : int, optional (default=None)
             Random seed to use.
 
-        sigma_decay_function : function (default=asymptotic_decay)
+        sigma_decay_function : string, optional
```

Comment:
All the possible values should be listed in the comment. Like here: https://github.com/JustGlowing/minisom/blob/master/minisom.py#L155